spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/pom.xml

```diff
@@ -16,4 +16,43 @@
 	<properties>
 		<main.basedir>${basedir}/../..</main.basedir>
 	</properties>
-</project>
\ No newline at end of file
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-gcp-core</artifactId>
+		</dependency>
+
+		<!-- Tests -->
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-test</artifactId>
+			<version>2.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot</artifactId>
+			<version>2.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
+			<version>5.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-beans</artifactId>
+			<version>5.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
+			<version>5.0.0.BUILD-SNAPSHOT</version>
```

Comment:
My concern that we don't position ourselves as Spring 5.0/Boot 2.0.
So, even if it works now, that doesn't guarantee that we won't meet some incompatibility in other place. 

So, if our `main` is Java 7 and SF-4.3/Boot-1.5 we can't make our tests based on different major versions.

I don't care about `TestPropertyValues` trap if we are going to fight with many other problems using Boot 2.0 in tests.

Please, try to find some other solution not using Boot 2.0 or even just SF 5.0.